### DISCUSSION TITLE
The data obtained by the get parameter is incorrect

### DISCUSSION THREAD

**Discussion by soloqi** [adonisjs/core#4503] [General]
### Package version

6.2.2

### Describe the bug

![err](https://github.com/adonisjs/core/assets/9653184/75ca37c9-88ab-4707-bb24-70f7aadef7f3)
![err1](https://github.com/adonisjs/core/assets/9653184/f5e88ae2-1cf9-4d0f-9423-1d415dd4e883)

get data incorrect  ！！！

**Comment by thetutlage** [comment]
@soloqi What do you expect and why?

**Reply by soloqi** [reply]
I expected the data received to be a string, but it is now being broken into an incorrect object. These do not have this problem in the v5 version. My requirement is to save the received parameters as they are for others to use without any additional processing of the data.
V6 print    ["{\"a\":\"1\"","\"b\":\"2\"}"]
V5 print    {"a":"1","b":"2"}

**Reply by holoyan** [reply]
Small test from my side


```typescript


router.get('/5', async ({ request }) => {
  console.log(typeof request.input('data')) // object

  // if data={"a":1} - ONE item (object?)string
  // then
  return request.input('data') // this is good

  // -----------------------------------------------------------------

  // if data={"a":1, "b":7} - more then one item
  // then
  return request.input('data') // this will return broken string
  // result
  /*
  [
    "{\"a\":1",
    " \"b\":5}"
]
   */
  // -----------------------------------------------------------------
  // if you call request.input('data').toString()
  // then everything is ok
  return request.input('data').toString() // {"a":1, "b":5}
})


```

**Reply by soloqi** [reply]
Thank you very much, this is what I want
